internal/schema/stacks/1.8/variable_block.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/terraform-schema/internal/schema/tokmod"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func variableBlockSchema() *schema.BlockSchema {
+	return &schema.BlockSchema{
+		SemanticTokenModifiers: lang.SemanticTokenModifiers{tokmod.Variable},
+		Labels: []*schema.LabelSchema{
+			{
+				Name:                   "name",
+				SemanticTokenModifiers: lang.SemanticTokenModifiers{tokmod.Name},
+				Description:            lang.PlainText("Variable Name"),
+			},
+		},
+		Description: lang.Markdown("Input variable allowing users to customizate aspects of the configuration when used directly " +
+			"(e.g. via CLI, `tfvars` file or via environment variables), or as a module (via `module` arguments)"),
```

Comment:
I think for stacks this needs some adjustments? Aren't they only set via the deployment block belonging to a stack?